src/start_fact.py

```diff
@@ -45,27 +45,28 @@ def _evaluate_optional_args(args):
 
 def _start_component(component, args):
     script_path = os.path.join(get_src_dir(), '../start_fact_{}'.format(component))
-    if os.path.exists(script_path):
-        logging.info('starting {}'.format(component))
-        optional_args = _evaluate_optional_args(args)
-        command = '{} -l {} -L {} -C {} {}'.format(
-            script_path, config['Logging']['logFile'], config['Logging']['logLevel'], args.config_file, optional_args
-        )
-        p = Popen(split(command))
-        return p
-    else:
+    if not os.path.exists(script_path):
```

Comment:
if we change the line, we should switch from os.path to Path, right?

Reply:
fixed